coding/wml-v20/src/main/java/org/n52/sos/aquarius/OmObservationMapper.java

```diff
@@ -97,4 +87,20 @@ public String toInterpolationTypeHref(final OmObservation observation) {
         return wml2InterpolationHref;
     }
 
+    private String mapProcedureIdentifierToInterpolationType(String procedureIdentifier) {
+        String aquariusInterpolationType = null;
+        if (null != procedureIdentifier && !procedureIdentifier.isEmpty()) {
+            int indexOfSeparationCharacter = procedureIdentifier.lastIndexOf(SEPARATOR_CHARACTER);
+            if (indexOfSeparationCharacter > 0) {
```

Comment:
The idea here is that if the id is "_abcd", "abcd" is probably not the interpolation type.  I'm saying there has to be at least one char in front of the "_" to get an interp type from the id.